coalib/settings/SectionFilling.py

```diff
@@ -45,7 +47,12 @@ def fill_section(section, acquire_settings, log_printer, bears):
 
     # Get missing ones.
     if len(needed_settings) > 0:
-        new_vals = acquire_settings(None, needed_settings, section)
+        if len(signature(acquire_settings).parameters) == 2:
+            logging.warning('acquire_settings: section parameter is '
+                            'deprecated.')
+            new_vals = acquire_settings(None, needed_settings)
+        else:
+            new_vals = acquire_settings(None, needed_settings, section)
```

Comment:
optional: maybe add an empty line here to separate the if a bit better from the `for`-loop below